src/Datadog.Trace.ClrProfiler.Managed/GlobalSuppressions.cs

```diff
@@ -4,3 +4,4 @@
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1119:Statement must not use unnecessary parenthesis", Justification = "ActuallyNeeded", Scope = "member", Target = "~M:Datadog.Trace.ClrProfiler.Integrations.AspNetIntegration.TryGetContext(System.Object,System.Web.HttpContext@)~System.Boolean")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1119:Statement must not use unnecessary parenthesis", Justification = "ActuallyNeeded", Scope = "member", Target = "~M:Datadog.Trace.ClrProfiler.Integrations.AspNetHttpModule.TryGetContext(System.Object,System.Web.HttpContext@)~System.Boolean")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.MaintainabilityRules", "SA1119:Statement must not use unnecessary parenthesis", Justification = "ActuallyNeeded", Scope = "member", Target = "~M:Datadog.Trace.ClrProfiler.Integrations.AspNetIntegration.Init(System.Object)")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.ReadabilityRules", "SA1118:Parameter must not span multiple lines", Justification = "ItShould", Scope = "member", Target = "~M:Datadog.Trace.ClrProfiler.Services.DependencyFactory.#cctor")]
```

Comment:
`GlobalSuppressions.cs` is meant for... well, global suppressions 🙂. The ones you have added have a specific target (`~M:Datadog.Trace.ClrProfiler.Services.DependencyFactory.#cctor`), so you added `SA1119` several times. If you want them to be really global, don't set the `Target` property. If your intention was to disable the check in a specific method, you can set the attribute on that method directly instead of doing it at the assembly level. Note also that we have shared cascading `GlobalSuppressions.cs` files, so it's easy to apply suppressions across multiple projects.